snapcraft/cli/assertions.py

```diff
@@ -151,6 +153,57 @@ def edit_collaborators(snap_name, key_name):
             echo.warning("The collaborators for this snap have not been altered.")
 
 
+@assertionscli.command("list-validation-sets")
+@click.option(
+    "--name",
+    metavar="<name>",
+    help="Only show results for a given Validation Set name.",
+)
+@click.option("--sequence", metavar="<sequence>", help="Sequences to show.")
+def list_validation_sets(name, sequence):
+    """Get the list of validation sets.
+
+    The sequence option can be a sequence number or a keyword.
+
+    \b
+    Examples:
+        snapcraft list-validation-sets
+        snapcraft list-validation-sets --sequence all
```

Comment:
Is `--sequence all` the same thing as no --sequence? Seems to be a bit odd. 

Not documented at below link either.

Reply:
Should be in below link as it is a store side thing and might evolve, @degville can we add "all" and "latest" (current default) to the docs?

Reply:
Yes, of course. I'll do that now - thanks!